app/models/pre_commencement_condition_validation_request.rb

```diff
@@ -4,6 +4,7 @@ class PreCommencementConditionValidationRequest < ValidationRequest
   RESPONSE_TIME_IN_DAYS = 10
 
   validates :cancel_reason, presence: true, if: :cancelled?
+  validates :condition_cancelled_at_now, presence: true, if: :cancelled?
```

Comment:
Should this be a `before_save` or `before_validation` hook instead of a validation? I think a validation should only return true or false rather than making a change.

Reply:
I've used the same pattern as line above. However it could also be a `before_validation`.

Reply:
It's not the same: `cancel_reason` is a database field, not a method. The `validates :cancel_reason` just checks that it's present; it doesn't set it to anything.